src/main/webapp/script.js

```diff
@@ -125,9 +111,48 @@ async function displayFeed() {
 
         displayRepAnswer(post, repName);
         displayReplyList(post);
+
+        //show reaction buttons 
+        var reactionDiv = document.createElement("div");
+        displayReaction(post, repName, reactionDiv, "ANGRY"); 
+        displayReaction(post, repName, reactionDiv, "CRYING"); 
+        displayReaction(post, repName, reactionDiv, "LAUGHING"); 
+        displayReaction(post, repName, reactionDiv, "HEART"); 
+        displayReaction(post, repName, reactionDiv, "THUMBS_UP"); 
+        displayReaction(post, repName, reactionDiv, "THUMBS_DOWN"); 
+        document.getElementById(post.id).appendChild(reactionDiv);
     })
 };
 
+function displayReaction(post, repName, reactionDiv, reaction) {
+    var btn = document.createElement("button");
+    btn.setAttribute("class", "btn"); 
+    var imageSrc = "reaction_icons/" + reaction.toLowerCase() + ".jpg"
+    btn.innerHTML = '<img src="'+ imageSrc +'" width="20px" height="20px">';
```

Comment:
Instead of concatenating strings here's another option that might be cleaner: 
var imageElement = document.createElement("img");
imageElement.src = "reaction_icons/" + reaction.toLowerCase() + ".jpg";
imageElement.style = "width="20px height="20px"
btn.apppendChild(imageElement);

Reply:
Whenever I try it this way and don't set the inner HTML, the reaction img doesn't show up at all.